oxidation/libparsec/crates/platform_local_db/src/executor.rs

```diff
@@ -27,7 +27,12 @@ impl SqliteExecutor {
             job_receiver,
             connection,
         };
-        let handle = std::thread::spawn(move || background_executor.serve());
+        // TODO: currently if the thread panic the error is printed to stderr,
+        // we should instead have a proper panic handler that log an error
+        let handle = std::thread::Builder::new()
+            .name("SqliteExecutor".to_string())
+            .spawn(move || background_executor.serve())
+            .expect("Cannot start thread");
```

Comment:
I don't think this is needed given `thread::spawn` does an expect:
https://doc.rust-lang.org/src/std/thread/mod.rs.html#702-709

Failing to create a thread means the OS is very angry at us and we are basically doomed, so doing an expect here seems the right thing to do given otherwise the caller will have to handle a weird corner case error that it will most likely be dealt with with a panic anyway ;-)

Reply:
Ok that